File: oop_lab/lab2/four.cpp
#include<iostream>

using namespace std;

int main(){
    int arr[100];
    int n;
    cin >> n;
    for(int i = 0; i < n; i++){
        cin >> arr[i];
    }
    int greatest = arr[0], smallest = arr[0];
    for(int i = 0; i < n; i++){
        if(greatest < arr[i]){
            greatest = arr[i];
        }
        if(smallest > arr[i]){
            smallest = arr[i];
        }
    }

    printf("Greatest no. : %d\nSmallest no. : %d\n", greatest, smallest);
    

}

File: oop_lab/lab2/four.kt
//finding the smallest and largest element in the array
fun main() {
    val arr = IntArray(20){(1..100).random()}
    arr.forEach { index -> print("$index\t")}
    var greatest = arr[0]
    var smallest = arr[0]
    arr.forEach {
        if(it < smallest){smallest = it}
        if(it > greatest){greatest = it}
    }
    println("\nGreatest element is $greatest")
    println("Smallest element is $smallest")
}